CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-### 4.10.5
+### 4.10.6
```

Comment:
You need to add the new changes for the April month. And keep the march changelog as well

Reply:
Ok, now it is done I think.